package.json

```diff
@@ -22,8 +22,8 @@
     "makeMigration": "knex migrate:make",
     "rollback": "knex migrate:rollback",
     "rollback-one": "knex migrate:down",
-    "start": "node dist/index.js",
-    "test": "TEST_ENVIRONMENT=true ts-mocha --require @babel/register --exit test/**",
+    "start": "node -r tsconfig-paths/register dist/index.js",
```

Comment:
When using `node` (instead of `ts-node`) shouldn't we add `-r ts-node/register/transpile-only` too?
If yes, this should be done everywhere node is used.